src/Loader/__snapshots__/Loader.test.js.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do you need snap files for the tests?

Reply:
@juancarlosfarah I think it is the "only" way to test standalone components? That way we can easily test for default parameters and defined parameters. Let me know if you are thinking about something else

Reply:
@pyphilia, but you don't need the snap file of the test file itself. It's usually a snap file of the component file (e.g. `Loader.js.snap`). Or am I wrong?